templates/AureliaSpa/Views/Shared/_Layout.cshtml

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>@ViewData["Title"] - Aurelia</title>
+        <base href="/" />
+    </head>
+    <body aurelia-app="boot">
```

Comment:
Is it definitely correct to have an `aurelia-app` attribute? There's a discussion at https://github.com/aurelia/bootstrapper-webpack/issues/2 that implies (for reasons I don't follow) that when you're using Webpack, you don't need `aurelia-app`. @EisenbergEffect, could you comment on whether this app is bootstrapping itself in the desired way?